<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigForm</class>
 <widget class="QWidget" name="ConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>方正黑体简体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>代理设置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ifProxyCheckBox">
       <property name="text">
        <string>启用代理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testProxyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>测试代理</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>代理端口</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="proxyPortLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>代理IP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="proxyIpLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>方正黑体简体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>引用JS文件夹设置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ifExternalJSCheckBox">
       <property name="text">
        <string>启用引用JS</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>引用JS文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="externalJSFolderPathLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectExternalJSFolderButton">
       <property name="text">
        <string>选择文件夹</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ifProxyCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ConfigForm</receiver>
   <slot>updateConnectButtonStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>testProxyButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigForm</receiver>
   <slot>connectProxy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigForm</receiver>
   <slot>saveConf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectExternalJSFolderButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigForm</receiver>
   <slot>selectExternalJSFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ifExternalJSCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ConfigForm</receiver>
   <slot>updateExternalJSStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateConnectButtonStatus()</slot>
  <slot>connectProxy()</slot>
  <slot>saveConf()</slot>
  <slot>selectExternalJSFolder()</slot>
  <slot>updateExternalJSStatus()</slot>
 </slots>
</ui>
